keyboards/squarebuilt/readme.md

```diff
@@ -0,0 +1,15 @@
+# squarebuilt
+
+![squarebuilt](imgur.com image replace me!)
```

Comment:
You should put an image URL here, if you have one. Otherwise, you can delete this line.